include/BigIntCallback.hpp

```diff
@@ -0,0 +1,36 @@
+// AUTOGENERATED FILE - DO NOT MODIFY!
+// This file generated by Djinni from callback.djinni
+
+#ifndef DJINNI_GENERATED_BIGINTCALLBACK_HPP
+#define DJINNI_GENERATED_BIGINTCALLBACK_HPP
+
+#include "../utils/optional.hpp"
+#include <memory>
+#ifndef LIBCORE_EXPORT
+    #if defined(_MSC_VER)
```

Comment:
[FAR] I think __dllexport/__dllimport is platform things, not a compiler. So we need to export/import for MinGW as well http://mingw.org/wiki/sampledll. Better check would be `#ifdef (WIN32)`

Reply:
This should be definitely replaced, when done it was only to support MSVC !